gulp/tasks/scripts.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+/* browserify task
+   ---------------
+   Bundle javascripty things with browserify!
+   This task is set up to generate multiple separate bundles, from
+   different sources, and to use Watchify when run from the default task.
+   See browserify.bundleConfigs in gulp/config.js
+*/
+
+var gulp = require( 'gulp' );
+var browserify = require( 'browserify' );
+var browserSync = require( 'browser-sync' );
+var watchify = require( 'watchify' );
+var mergeStream = require( 'merge-stream' );
+var bundleLogger = require( '../utils/bundleLogger' );
+var handleErrors = require( '../utils/handleErrors' );
+var source = require( 'vinyl-source-stream' );
+var config = require( '../config' ).scripts;
```

Comment:
Should be `var scripts`, see https://nodejs.org/api/child_process.html for example of naming style.